### DISCUSSION TITLE
Tray app without WebView processes when hidden

### DISCUSSION THREAD

**Discussion by TheFranconianCoder** [tauri-apps/tauri#6308] [Q&A] [answered]
Hi, when I build an app that mostly sits in the tray, only hiding the WebView still costs this ressources. Is it possible to start the WebView processes only on demand, when any action triggers it.

**Comment by FabianLars** [comment, marked as answer]
You need to close the webview _window_, and create the window on demand (like on tray menu events) - both of course in Rust because without a webview there is no JS process/runtime.

you can keep the app running without a window like this: <https://github.com/tauri-apps/tauri/blob/b02fc90f450ff9e9d8a35ee55dc1beced4957869/examples/api/src-tauri/src/main.rs#L240-L244>

**Comment by TheFranconianCoder** [comment]
@FabianLars Thank you! Will check this out. Exactly what I was looking for.
